Droid/GiTracker.Droid.csproj

```diff
@@ -13,7 +13,7 @@
     <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
     <AndroidApplication>True</AndroidApplication>
     <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
-    <AssemblyName>GiTracker.Droid</AssemblyName>
+    <AssemblyName>GiTracker.App</AssemblyName>
```

Comment:
In order to have common assembly name for Droid and iOS project to be able to write this:

<pages:BasePage 
   ...
   xmlns:pages="clr-namespace:GiTracker.Views;assembly=GiTracker.App"
   ...>